app/templates/suppliers/edit_what_buyers_will_see.html

```diff
@@ -0,0 +1,106 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/forms/macros/forms.html" as forms %}
+
+{% block page_title %}What buyers will see – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account"
+      },
+      {
+        "link": url_for(".supplier_details"),
+        "label": "Company details"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+{% if error %}
+<div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+  <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+    {{ error }}
+  </h3>
+</div>
+{% endif %}
+{% if supplier_form.errors or contact_form.errors %}
+    <div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+        <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+            There was a problem with the details you gave for:
+        </h3>
+        {% if supplier_form.errors %}
+        <ul>
+        {% for field_name, field_errors in supplier_form.errors|dictsort %}
+          {% for error in field_errors %}
+            <li><a href="#{{ field_name }}" class="validation-masthead-link">{{ supplier_form[field_name].label.text }}</a>
+          {% endfor %}
+        {% endfor %}
+        </ul>
+        {% endif %}
+
+        {% if contact_form.errors %}
+        <ul>
+        {% for field_name, field_errors in contact_form.errors|dictsort %}
+          {% for error in field_errors %}
+          <li><a href="#contact_{{ field_name }}" class="validation-masthead-link">{{ contact_form[field_name].label.text }}</a></li>
+          {% endfor %}
+        {% endfor %}
+        </ul>
+        {% endif %}
+    </div>
+{% endif %}
+
+  <div class="grid-row">
+    <div class="column-one-whole">
+      {% with
+        heading = "What buyers will see"
+      %}
+        {% include 'toolkit/page-heading.html' %}
+      {% endwith %}
+
+      <div class="dmspeak">
+        <p>This information will be visible on the Digital Marketplace.</p>
+        <p>You can change it at any time.</p>
+      </div>
+    </div>
+  </div>
+
+  <form action="{{ url_for('.edit_what_buyers_will_see') }}" method="post" enctype="multipart/form-data">
```

Comment:
You don't need the `enctype` attribute here, only if you're uploading a file. David linked me to a good article about it when he was reviewing the country picker. https://stackoverflow.com/questions/4526273/what-does-enctype-multipart-form-data-mean

Reply:
Ok, I will get rid of it 👍